packages/typescript/internal/build_defs.bzl

```diff
@@ -51,8 +51,6 @@ def _trim_package_node_modules(package_name):
         segments += [n]
     return "/".join(segments)
 
-# This function is similar but slightly different than _compute_node_modules_root
-# in /internal/node/node.bzl. TODO(gregmagolan): consolidate these functions
```

Comment:
haha after all the duplication we don't want to commit to reducing it :)

Reply:
Yep 🤷 . This ts_library impl is no longer reducible as it does the old single workspace check & not the new single workspace / path check.